doc/tutorials/mlip/mlip.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<ul><li>We haven't seen the first parts yet. Up to now, I don't know if it has accurate forces or can run MD. There is no comparison. You can do a classical force-field comparison, for example, or show DFT data. I actually have DFT ethanol trajectories you can use. Or, run a short DFT ethanol MD with PySCF, it is pretty fast.</li><li>Maybe list some chemical reaction methods.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4982/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>